Support_Page.php

```diff
@@ -59,13 +59,13 @@ static public function admin_print_scripts_w3tc_support() {
 	 * @return void
 	 */
 	function options() {
-		if ( isset( $_GET['done'] ) ) {
+		if ( isset( Util_Request::get_string( 'done' ) ) ) {
 			$postprocess_url =
 				'admin.php?page=w3tc_support&w3tc_support_send_details' .
-				'&_wpnonce=' . urlencode( $_GET['_wpnonce'] );
+				'&_wpnonce=' . urlencode( Util_Request::get_string( '_wpnonce' ) );
 			foreach ( $_GET as $p => $v ) {
 				if ( $p != 'page' && $p != '_wpnonce' && $p != 'done' )
-					$postprocess_url .= '&' . urlencode( $p ) . '=' . urlencode( $v );
+					$postprocess_url .= '&' . urlencode( $p ) . '=' . urlencode( Util_Request::get( $p ) );
```

Comment:
Use `Util_Request::get_string()`.

Reply:
Added missing Util_Request::get_string and refactored some other code for better optimization